dev-python/ipywidgets/ipywidgets-9999.ebuild

```diff
@@ -25,11 +25,9 @@ IUSE="test"
 RDEPEND="
 	dev-python/traitlets[${PYTHON_USEDEP}]
 	dev-python/ipykernel[${PYTHON_USEDEP}]
-	$(python_gen_cond_dep 'dev-python/wxpython:*[${PYTHON_USEDEP}]' python2_7)
 	"
 DEPEND="${RDEPEND}
 	test? (
-		$(python_gen_cond_dep 'dev-python/mock[${PYTHON_USEDEP}]' python2_7)
```

Comment:
Mock is needed for Python 2.7 to run the tests, I don't think this line should be removed

https://github.com/ipython/ipywidgets/blob/master/ipywidgets/widgets/tests/test_interaction.py#L11